server/game/cards/characters/01/thequeenofthorns.js

```diff
@@ -6,24 +6,17 @@ class TheQueenOfThorns extends DrawCard {
             when: {
                 afterChallenge: (event, challenge) => challenge.winner === this.controller && challenge.isParticipating(this) && challenge.challengeType === 'intrigue'
             },
-            handler: () => {
-                this.game.promptForSelect(this.controller, {
-                    activePromptTitle: 'Select a character to put in play',
-                    source: this,
-                    cardCondition: card => card.location === 'hand' && card.getCost() && card.getType() === 'character' && card.isFaction('tyrell'),
-                    onSelect: (p, card) => this.onCardSelected(p, card)
-                });
+            target: {
+                activePromptTitle: 'Select a character to put into play',
+                cardCondition: card => card.location === 'hand' && card.controller === this.controller &&
```

Comment:
"card.controller === this.controller" shouldn't be needed, given the card is in hand, right?
I guess there could be bugs making that assumption false. But, in that case too, I'd rather not have the check because it will hide the bugs.